lvsfunc/util.py

```diff
@@ -59,6 +56,9 @@ def quick_resample(clip: vs.VideoNode,
 
     :return:            Filtered clip in original depth.
     """
+    warnings.warn("quick_resample: 'This function will be either reworked or removed in a future version!'",
```

Comment:
💣 💣 💣 
   Nuke it

Reply:
Probably next version. I want to double-check where it's being used first and make sure people change it lol